paywall/src/unlockTypes.ts

```diff
@@ -102,6 +107,6 @@ export interface Key {
   transactions: Transaction[]
   status: string
   confirmations: number
-  owner: string
+  owner: string | null
```

Comment:
In which cases do keys not have owners?

Reply:
When the user has no wallet whatsoever. In that case the key is a dummy value used only for display of the lock